visualization/settings/config.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional, Tuple
+
+import hydra
+
+from nemo_skills.code_execution.utils import CODE_OUTPUT_SEPARATORS, CODE_SEPARATORS
+from nemo_skills.inference.generate_solutions import GenerateSolutionsConfig
+from nemo_skills.utils import unroll_files
+
+
+@dataclass
+class BaseVisualizationConfig:
+    model_prediction: Dict[str, str] = field(default_factory=dict)
```

Comment:
why do you need a default to empty dict? Shouldn't it be MISSING instead?

Reply:
The "run model" page doesn't need this parameter, so I don't want to force it to be filled in